cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -11,65 +11,49 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
 [[RN_new_features]]
 == New and changed features
 
-==== Virtual machine control from the web console
-** The updated {ProductName} web console now allows users to create virtual 
-machines, access virtual machines with either a serial or VNC connection, and 
-perform basic power management tasks (start|stop) on virtual machines. 
-
-==== Layer-2 networking
-** Layer-2 networking capabilities are now exposed to the {ProductName} 
-web console. Users can control the type of vNIC that is exposed to a virtual 
-machine, connect a virtual machine to an enterprise network defined by a VLAN 
-ID, and more.
+==== Updated KubeVirt API
+* A new version of the KubeVirt API is included in {ProductName} 1.4. Several 
+important changes are reflected in the latest configuration file templates.
+** The `apiVersion` has been updated from `kubevirt.io/v1alpha2` to 
+`kubevirt.io/v1alpha3`.
+** The `volumeName` attribute no longer exists. Ensure that each disk name 
+matches the corresponding volume name in all configuration files.
+** All instances of `registryDisk` must be updated to `containerDisk` in
+configuration files.
+
```

Comment:
```
=== Introducing Operators
** The new install of KubeVirt, Containerized Data Imported, and web-ui is powered by Operators.
```

Reply:
Thanks Ryan, check my latest update. I reworded it slightly:

=== Introducing Operators
* Operators power the installation of KubeVirt, Containerized Data Importer, and
the *web-ui* in {ProductName} 1.4.